ProjectSidewalk/SidewalkWebpage
# Resolves #1648: Logging when unloading the page

Resolves #1648 

Added two endpoints ```/taskBeacon``` and ```/validationTaskBeacon``` that accept plaintext JSON and simulate a redirect to ```/task``` and ```/validationTask``` respectively with the plaintext converted to "actual" JSON format.

The beacons on the client side now use these endpoints since ```navigator.sendBeacon``` doesn't support ```application/json```.

To test:
First try to reproduce this on develop:
* comment out @misaugstad 's temporary fix in "public/..../SVLabel/..../Form.js" since that's what we're trying to replace. The line to comment out is the self.submitData line after the "TODO: Replace with beacon (or some ajax alternative) asap. Starter code below" comment
* make sure that "Unload" events (and other events close to a page refresh) aren't being recorded in the **audit_task_interaction** and **validation_task_interaction** tables

Now test on this branch:
- Check that "Unload" events are being recorded in the **audit_task_interaction** and **validation_task_interaction** tables
- Check that labels/validations done right before a refresh save
